client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/test/task-helper.ts

```diff
@@ -91,43 +91,40 @@ describe( 'Task Helpers', () => {
 		} );
 	} );
 
```

Comment:
The testing updates below are intended just to be sure this PR doesn't break tests. We want them to pass. But I'm really on testing if the domain upsell task is included in flows. In a follow up, we may want to add tests to confirm the task is complete/incomplete based on free/paid status.

Reply:
> In a follow up, we may want to add tests to confirm the task is complete/incomplete based on free/paid status.

Can we create a follow-up issue?

Reply:
Yup, was just doing there. Here it is: https://github.com/Automattic/wp-calypso/issues/73777